frontend/src/metabase/admin/people/selectors.js

```diff
@@ -31,3 +34,14 @@ export const getUserMemberships = createSelector(
 
 export const getUserTemporaryPassword = (state, props) =>
   state.admin.people.temporaryPasswords[props.userId];
+
+export const shouldNudgeToPro = createSelector(
+  state => getSetting(state, "premium-embedding-token"),
```

Comment:
@alxnddr Is this the best way to detect pro/enterprise?

Reply:
@shaunlebron you can check for `token-features` because our Starter cloud instances assign tokens although users don't have this `sso` token feature which we are trying to promote

<img width="754" alt="Screen Shot 2023-06-19 at 2 10 31 PM" src="https://github.com/metabase/metabase/assets/14301985/fb9c85fc-ae44-4878-9441-49b8fb63f56f">